draft-ietf-webtrans-http2.md

```diff
@@ -203,10 +203,16 @@ of !SEMANTICS=I-D.ietf-httpbis-semantics}}. The WebTransport server MUST verify
 the `Origin` header to ensure that the specified origin is allowed to access
 the server in question.
 
-From the client's perspective, a WebTransport session is established when the
-client receives a 200 response. From the server's perspective, a session is
-established once it sends a 200 response. Both endpoints MUST NOT send any
-WebTransport capsules on a given session before that session is established.
+A WebTransport session is established when the server sends a 200 response. A
+server generates that response from the request header, not from the contents
+of the request. To enable clients to resend data when attempting to
+re-establish a session that was rejected by a server, a server MUST NOT process
+any capsules on the request stream unless it accepts the WebTransport session.
+A client MAY optimistically send any WebTransport capsules associated with a
```

Comment:
```suggestion

A client MAY optimistically send any WebTransport capsules associated with a
```

Looking a bit wall-of-text-like.  Not sure if this breaks the flow too much.  It doesn't seem to.

Reply:
Looking good 👍